src/gui/addnewtorrentdialog.cpp

```diff
@@ -300,6 +300,9 @@ AddNewTorrentDialog::AddNewTorrentDialog(const BitTorrent::AddTorrentParams &inP
     // TODO: set dialog file properties using m_torrentParams.filePriorities
     m_ui->setupUi(this);
 
+    connect(m_ui->buttonBox, &QDialogButtonBox::accepted, this, &AddNewTorrentDialog::accept);
```

Comment:
I would use base slots in such "regular" connections like this one (and `reject` as well):
```suggestion
    connect(m_ui->buttonBox, &QDialogButtonBox::accepted, this, &QDialog::accept);
```

Reply:
What if the class override `QDialog::accept()`? then it would be wrong to connect to the base class slot.

Reply:
Those slots are virtual.

Reply:
Done.